examples/installation/build.sh

```diff
@@ -6,7 +6,6 @@ Usage: sh $0 command [argument]
 
 command:
   test-all:                        tests all projects in this repo.
-  test-xcode6:                     tests all Xcode 6 projects in this repo.
   test-xcode7:                     tests all Xcode 7 projects in this repo.
```

Comment:
looks like this actually drops support for Xcode 7, which is fine, as long as these names are updated to refer to Xcode 8.